payment-request/payment-response/onpayerdetailchange-attribute-manual.https.html

```diff
@@ -16,7 +16,14 @@
     await response.retry({ error });
     const event = await eventPromise;
     assert_true(event instanceof PaymentRequestUpdateEvent);
-    for(const [prop, value] of Object.entries(expected)){
+    for([prop, value] of Object.entries(expected)){
+      if (prop === 'payerPhone') {
+        // |payerPhone| may optionally adhere to E164 structure, which does not
+        // contain formatting, e.g. +180000000 instead of +1-800-000-0000.
+        // Strip out the formatting in case the user agent implements E164.
+        // https://w3c.github.io/payment-request/#addressinit-dictionary
+        value = value.replace(/-/g, '');
```

Comment:
Should we also ignore internal spaces and parentheses, so "+1 (800) 000-0000 "is also valid?

Reply:
Good idea. Done.